sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -17,7 +17,7 @@
 #ifdef __INTEL_PREVIEW_BREAKING_CHANGES
 #include <sycl/detail/string_view.hpp>
 #endif
-#include <sycl/device.hpp>                     // for device
+#include <sycl/device.hpp> // for device
```

Comment:
This formatting change doesn't seem to match the rest, are we sure it is correct?